src/transformers/pipelines.py

```diff
@@ -656,7 +656,7 @@ class TextClassificationPipeline(Pipeline):
 
     def __call__(self, *args, **kwargs):
         outputs = super().__call__(*args, **kwargs)
-        scores = np.exp(outputs) / np.exp(outputs).sum(-1)
+        scores = np.exp(outputs) / np.exp(outputs).sum(-1).reshape(-1,1)
```

Comment:
You can also just do 
```suggestion
        scores = np.exp(outputs) / np.exp(outputs).sum(-1, keepdims=True)
```

Reply:
Thanks for pointing this out!